ppaas/certificate.py

```diff
@@ -46,7 +46,7 @@ class Certificate(object):
             }
         }
     """
-    def __init__(self, master, hostname):
+    def __init__(self, master, hostname, cached_data=None):
```

Comment:
Plus update the docstring of the classes to specify the cached data will be used instead of making an HTTP request each time